apps/ledger-live-mobile/src/newArch/features/FirmwareUpdate/utils/isFirmwareUpdateSupported.ts

```diff
@@ -0,0 +1,40 @@
+import { Platform } from "react-native";
+import { DeviceModelInfo } from "@ledgerhq/types-live";
+import { DeviceModelId } from "@ledgerhq/devices";
+import isFirmwareUpdateVersionSupported from "@ledgerhq/live-common/hw/isFirmwareUpdateVersionSupported";
+import { lastConnectedDeviceSelector } from "~/reducers/settings";
+
+export function isNewFirmwareUpdateUxSupported(deviceModelId?: DeviceModelId) {
+  switch (deviceModelId) {
+    case DeviceModelId.stax:
+    case DeviceModelId.europa:
+      return true;
+    default:
+      return false;
+  }
+}
+
+export function isOldFirmwareUpdateUxSupported({
+  lastSeenDeviceModelInfo,
+  lastConnectedDevice,
+}: {
+  lastSeenDeviceModelInfo: DeviceModelInfo | null | undefined;
+  lastConnectedDevice: ReturnType<typeof lastConnectedDeviceSelector> | null | undefined;
```

Comment:
Do we want to explicitly pass `undefined`? Can't we make it optional with `?:`?

Reply:
The `undefined` type comes from the definition of the settings state (in the redux store), I'm going to check because it might actually be unnecessary (`null` being the default value).